### ISSUE TITLE
A simple way to replace siimple colors

### ISSUE THREAD

**Issue by valentinbdv** [siimple/siimple#49]
**Is your feature request related to a problem? Please describe.**
My problem is that I would like to be able to use my own colors 

**Describe the solution you'd like**
A simple way to replace siimple's default values 

**Describe alternatives you've considered**
The first obvious solution was to overwrite all colours and background for each class, but it can feel like you are rewriting the all siimple CSS.
Secondly, we could rename all the legacy colours and palette colours from the variables file but it will mean we will have the redo the operation at each package update.

**Additional context**
Thanks for your help. Really great library, like the fact that this is CSS only! ;)

**Comment by jmjuanes** [comment]
Hello @valentinbdv ! Thanks for your issue!

Yes, you can replace the default colours of siimple without rewriting the library. As you can see in file `scss/_variables.scss`, all the colours variables has a `!default` flag, so you can create a SCSS file, for example `custom-siimple.scss`, and overwrite the variables you want:

```scss
//My custom colors
$siimple-default-primary: #4e91e4;
$siimple-default-success: #4acf7f;
$siimple-default-warning: #fbc850;
$siimple-default-error: #ee675d;

//Load the siimple library
@import "siimple/scss/siimple.scss";
```
Now you can compile this file with sass, and you will get a CSS file with all the siimple components but using your own colors.

Please, try it and tell me if that solves your question. If not, we can find another solution :smile:

**Comment by valentinbdv** [comment]
@jmjuanes thanks a lot for your quick answer. Impressive!

It works, I indeed can easily change siimple's colours. 
But there is a drawback, now the siimple style is compiled after mine and so it overwrites my style when I want the opposite.
Not sure how I can avoid that while still being able to have my colours used by siimple.

**Comment by valentinbdv** [comment]
Ok I figured it out. 

Just had to import custom-siimple file at first. 

Thanks again ;)

**Comment by jmjuanes** [comment]
@valentinbdv You can import only your custom-siimple, because both have the same components. If you import the original and your custom version, you are loading the same library twice...

**Comment by valentinbdv** [comment]
Yes @jmjuanes I was importing it only once. 
But the issue was that I imported my custom-siimple (which import all siimple) after every CSS files. So I just had to import it first to allow my own style to overwrite siimple style.

**Comment by jmjuanes** [comment]
@valentinbdv Perfect! :smile:

**Comment by jmjuanes** [comment]
Closing this issue. Please, if you have more questions please open a new issue! Thank you :smile:
